08_files_lab/02.py

```diff
@@ -1,7 +1,18 @@
-"""
-Write a program that takes 3 names. The first two
-are existing file names and the last is a new file name
-Program should write the lines from the first two 
-files interwinded into the output file
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 2
+
+"""
+
+import sys
+import binascii
+from itertools import izip_longest
+
+with open(sys.argv[1], "r") as fin1:
+    with open(sys.argv[2], "r") as fin2:
+        with open(sys.argv[3], "w") as fout:
+            print >> fout, list(izip_longest(fin1,fin2,fillvalue=None))
```

Comment:
Hi,
Could you give more example about izip_longest function?

How can i fix my output?
![image](https://cloud.githubusercontent.com/assets/20268575/16761627/a53c4584-4829-11e6-8e77-3e175236eb15.png)